src/java/entities/User.java

```diff
@@ -13,8 +13,8 @@
 @Table(name="User",schema="storio")
 public abstract class User implements Serializable {
     
-     @Id
-     @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
please beware of committing changes to files you aren't assigned to, while this edit is super good it could cause unnecessary conflicts